.github/workflows/documentation.yaml

```diff
@@ -54,7 +54,8 @@ jobs:
         with:
           persist-credentials: false
       - name: Run pre-requisites for validation
-        make -C Documentation copy-api # necessary for check-build.sh
+        run: |
+          make -C Documentation copy-api # necessary for check-build.sh
```

Comment:
```suggestion
        run: make -C Documentation copy-api # necessary for check-build.sh
```

Reply:
We mostly have `run` blocks rather than one-liners in our workflows so I went for consistency, but I'm OK either way.